src/test/kotlin/com/esops/order/OrderTest.kt

```diff
@@ -0,0 +1,302 @@
+package com.esops.order
+
+import com.esops.e2e.CommonUtil
+import com.esops.entity.EsopType
+import com.esops.entity.OrderStatus
+import com.esops.entity.OrderType
+import com.esops.service.OrderService
+import com.esops.service.UserService
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import jakarta.inject.Inject
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.math.BigInteger
+
+@MicronautTest
+class OrderTest {
```

Comment:
Service tests are not micronaut tests. They need to be unit tests. You are doing something wrong if they are micronaut tests. 

It does not require a server, why do you have this annotation.

Reply:
Changed injection type